lib/reek/context_builder.rb

```diff
@@ -16,16 +18,16 @@ module Reek
   # counting. Ideally `ContextBuilder` would only build up the context tree and leave the
   # statement and reference counting to the contexts.
   #
-  # :reek:TooManyMethods: { max_methods: 27 }
+  # :reek:TooManyMethods: { max_methods: 30 }
   # :reek:UnusedPrivateMethod: { exclude: [ !ruby/regexp /process_/ ] }
   class ContextBuilder
     attr_reader :context_tree
-    private_attr_accessor :element
+    private_attr_accessor :current_context
```

Comment:
Oh, wait, I forgot "element" for my hitlist of bad naming in `Reek` :laughing: 
Love it that we fix this now as well.